client/state/reader/analytics/actions.js

```diff
@@ -1,12 +1,16 @@
+import { buildReaderTracksEventProps } from 'calypso/reader/stats';
 import { recordTracksEvent } from 'calypso/state/analytics/actions';
 import { getReaderFollowsCount } from 'calypso/state/reader/follows/selectors';
 
-export const recordReaderTracksEvent = ( name, properties ) => ( dispatch, getState ) => {
-	const followsCount = getReaderFollowsCount( getState() );
-	dispatch(
-		recordTracksEvent( name, {
-			subscription_count: followsCount,
-			...properties,
-		} )
-	);
-};
+export const recordReaderTracksEvent =
+	( name, properties, { pathnameOverride, post } = {} ) =>
+	( dispatch, getState ) => {
+		properties = buildReaderTracksEventProps( properties, pathnameOverride, post );
+		const followsCount = getReaderFollowsCount( getState() );
+		dispatch(
+			recordTracksEvent( name, {
+				subscription_count: followsCount,
+				...properties,
+			} )
+		);
```

Comment:
Nit: I'd prefer we don't unnecessarily (re)use `properties` when the actual data we're logging will always be modified by `buildReaderTracksEventProps()`. For me, it would feel clearer and a bit cleaner if we used a dedicated variable for the returned data.
```suggestion
	( name, properties, { pathnameOverride, post } = {} ) =>
	( dispatch, getState ) => {
		const eventProps = buildReaderTracksEventProps( properties, pathnameOverride, post );
		const followsCount = getReaderFollowsCount( getState() );
		dispatch(
			recordTracksEvent( name, {
				subscription_count: followsCount,
				...eventProps,
			} )
		);
```

Reply:
Im fine with that. Its how I would normally default to writing it, but surrounding/previous code here inspired me to do it this way for whatever reason. Im happy to commit your suggestions.